custom_components/birthdays/__init__.py

```diff
@@ -33,10 +39,13 @@ async def async_setup(hass, config):
     devices = []
 
     for birthday_data in config[DOMAIN]:
-        name = birthday_data[CONF_NAME]
-        date_of_birth = birthday_data[CONF_DATE_OF_BIRTH]
-        icon = birthday_data[CONF_ICON]
-        devices.append(BirthdayEntity(name, date_of_birth, icon, hass))
+        unique_id            = birthday_data[CONF_UNIQUE_ID]
+        name                 = birthday_data[CONF_NAME]
+        date_of_birth        = birthday_data[CONF_DATE_OF_BIRTH]
+        icon                 = birthday_data[CONF_ICON]
+        attributes           = birthday_data[CONF_ATTRIBUTES]
+        age_at_next_birthday = birthday_data[CONF_AGE_AT_NEXT_BIRTHDAY]
```

Comment:
This can also be removed, since line 30 is removed. Same with passing it on to the constructor

Reply:
Removed!